bin/ch/ch.cpp

```diff
@@ -402,13 +405,23 @@ HRESULT CreateAndRunSerializedScript(const char* fileName, LPCSTR fileContents,
     IfJsErrorFailLog(ChakraRTInterface::JsGetCurrentContext(&current));
     IfJsErrorFailLog(ChakraRTInterface::JsSetCurrentContext(context));
 
+    // canonicalize that path name to lower case for the profile storage
+    // REVIEW: Assuming no utf8 characters here
```

Comment:
how much effort would it be to handle unicode characters?  this seems like it could easily bite someone in the future, particularly folks in non-english speaking regions